app/src/main/java/com/example/myfridgehome/dto/StoredFoods.kt

```diff
@@ -1,6 +1,8 @@
 package com.example.myfridgehome.dto
 
-data class StoredFoods(var id : Int, var name: String, var type: String, var quantity : Int, var measurement : String){
+import com.google.gson.annotations.SerializedName
+
+data class StoredFoods ( @SerializedName ("id") var id : Int, @SerializedName("name")var name: String, @SerializedName("type") var type: String, @SerializedName("quantity") var quantity : Int, @SerializedName("measurment") var measurement : String){
```

Comment:
@SerializedName is an annotation that indicates the annotated member should be serialized to JSON with the provided name value as its field name. It will give nice formatting.

Reply:
Ditto comments above.